### PR TITLE
Reserve enums used by cl_ext_yuv_images

### PR DESCRIPTION
**Pull request by kpet**
Covers all enums currently used by #722.

### CODE REVIEW

File: xml/cl.xml [hunk 1]
```diff
@@ -899,6 +899,7 @@ server's OpenCL/api-docs repository.
         <enum bitpos="1"            name="CL_MAP_WRITE"/>
         <enum bitpos="2"            name="CL_MAP_WRITE_INVALIDATE_REGION"/>
             <unused start="3" end="31"/>
+            <unused start="32" end="33" comment="Reserved for cl_ext_yuv_images #722"/>
```
Comment (alycm) [inline xml/cl.xml:902]: If we're going beyond 32-bits should there be an extra line saying

```c
<unused start="34" end="63"/>
```

`cl_map_flags` is `cl_bitfield`, which is `cl_ulong`, so 64-bits are available.

**Review by alycm — APPROVED** [review]
1 small question, fine in principle.

File: xml/cl.xml (hunk 1 above)
Comment (kpet) [reply xml/cl.xml:902]: That makes sense. I've added the missing unused range.

**Review by Jeremy-Kemp — APPROVED** [review]
LGTM!

**Comment by kpet** [comment]
Merging with two approvals.

**Pull request merged**